zds/tutorialv2/models/database.py

```diff
@@ -1515,4 +1516,27 @@ def transfer_paternity_receiver(sender, instance, **kwargs):
     PublishedContent.objects.transfer_paternity(instance, external)
 
 
+class Clap(models.Model):
+    class Meta:
+        verbose_name = "Clap"
+        verbose_name_plural = "Claps"
+
+    content = models.ForeignKey(
+        PublishableContent,
+        verbose_name="Contenu",
+        on_delete=models.CASCADE,
+        related_name="claps",
+        db_index=True,
+    )
+    user = models.ForeignKey(User, db_index=True, on_delete=models.SET_NULL, null=True)
+    hash_ip_address = models.CharField(db_index=True, max_length=40)
```

Comment:
`hash_ip` retourne une chaîne de 64 caractères maximum
```suggestion
    hash_ip_address = models.CharField(db_index=True, max_length=64)
```

Reply:
Alors du coup, ça se passe comment pour les migrations ? Si je les regénère classiquement, ça va faire 2 fichiers de migrations, non ?

Reply:
Oui, le mieux est probablement de supprimer le fichier de migration actuel et de refaire `python manage.py makemigrations`.

Reply:
D'accord, merci. Et ça ne risque pas de coincer avec la BDD que j'ai en l'état ?

Reply:
Peut-être oui, auquel cas il faudra lancer `make new-db` pour en avoir une neuve.